packages/curve-ui-kit/src/shared/ui/PositionDetails/tooltips/LiqThresholdMetricTooltip.tsx

```diff
@@ -0,0 +1,43 @@
+import { Stack, Typography } from '@mui/material'
+import { FORMAT_OPTIONS, formatNumber } from '@ui/utils/utilsFormat'
+import { t } from '@ui-kit/lib/i18n'
+import { BandRange, LiquidationRange } from '@ui-kit/shared/ui/PositionDetails'
+import { SizesAndSpaces } from '@ui-kit/themes/design/1_sizes_spaces'
+
+const { Spacing } = SizesAndSpaces
+
+type LiqThresholdTooltipProps = {
+  liquidationRange: LiquidationRange | undefined | null
+  bandRange: BandRange | undefined | null
+}
+
+export const LiqThresholdTooltip = ({ liquidationRange, bandRange }: LiqThresholdTooltipProps) => (
+  <Stack gap={3} sx={{ maxWidth: '20rem' }}>
+    <Typography variant="bodySRegular">{t`The price at which your position enters liquidation protechtion and your collateral starts to be eroded.`}</Typography>
+
+    <Stack gap={2} display="column" sx={{ backgroundColor: (t) => t.design.Layer[2].Fill, padding: Spacing.sm }}>
+      <Typography variant="bodySBold">{t`Breakdown`}</Typography>
+
+      <Stack direction="row" justifyContent="space-between" gap={5}>
+        <Typography variant="bodySRegular">{t`Liquidation threshold`}</Typography>
+        <Typography variant="bodySBold">
+          {liquidationRange?.value?.[1] ? formatNumber(liquidationRange.value[1], { ...FORMAT_OPTIONS.USD }) : '-'}
```

Comment:
All these `'-'` everywhere will make it impossible to change the notation consistently. They should be part of some helpers